hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -1363,24 +1370,17 @@ public Response saveAllJobs(@QueryParam("user") String user,  @QueryParam("token
             log.trace("Save all jobs exception", ex);
             return Response.ok("{\"operation\":\"failed: " + ex.toString() + "\"").build();
         }
-
     }
 
     private static Response buildServerError(Exception exception) {
-        log.warn("", exception);
+        log.error("Error in kick job: {}", Throwables.getStackTraceAsString(exception));
         String message = exception.getMessage();
         if (message == null) {
             message = exception.toString();
         }
-
-        String stack = Throwables.getStackTraceAsString(exception);
-
-        final String response = "{" +
-                "\"error\": \"A java exception was thrown." +
-                "\"message\": \"" + StringEscapeUtils.escapeEcmaScript(message) + "\"" +
-                "\"stack\": \"" + StringEscapeUtils.escapeEcmaScript(stack) + "\"" +
-                "\"}";
-
+        final String response =
+                "ERROR: " + exception.getClass() + " was thrown <br/>" +
+                "MESSAGE: " + StringEscapeUtils.escapeEcmaScript(message);
```

Comment:
`buildServerError` is used in many places. Have you done a proper audit of all the js code where the returned server error is used? If not, the change from json to plain text can potentially break front end things. A safer change is to keep the json format, but remove the `stack` element - the stack trace would almost certainly mess up fron end display and should not be shown to the end user anyway - so this should be an improvement